stories/AccountStatus/AccountStatus.stories.svelte

```diff
@@ -9,85 +9,87 @@
 <Meta title="Example/Profile" component={AccountStatus} />
 
 <Story name="Status">
-  <section>
-    <div class="h4">Anonymous user</div>
-    <div class="row v-center">
-      <AccountStatus />
-    </div>
-  </section>
+  <div class="column">
```

Comment:
`section` is a `block` element. There is no need to create additional nesting
```suggestion
```

Reply:
Fixed